deps/v8/src/api.cc

```diff
@@ -2227,6 +2227,10 @@ int PrimitiveArray::Length() const {
   return array->length();
 }
 
+void PrimitiveArray::Set(Isolate* isolate, int index, Local<Primitive> item) {
```

Comment:
> This is problematic because it means prebuilt add-ons that call these methods won't work with older Node.js v10.x releases that don't have them.

I mean, that’s a general issue with semver-minor patches for the C++ API, right? I’m not too worried here. If you think we should do something, I guess we could include the Node.js version number in the warning message?